pages/docs/tutorials/kafka/configure-kafka-avro.md

```diff
@@ -0,0 +1,145 @@
+
+## Background Context
+
+AsyncAPI is a specification for describing Event-Driven Architectures (EDAs) in a machine-readable format. It allows developers to define how messages are exchanged between different services by providing a agreement for communication.
+
+Avro is an efficient binary serialization format, used to ensure schema-aware communications between the messages within Apache Kafka. With Avro Schema there is a standardized method for serializing data allowing for interoperability and schema evolution capabilities. Avro promotes data exchange among systems by offering a common schema that fosters compatibility, between different components. 
+
+## Defining messages using Avro Schema in YAML
+
+In this section, let's focus on defining messages using Avro Schemas using YAML. 
+
+```
+messages:
+  userSignedUp:
+    payload:
+      schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+      schema:
+        type: record
+        name: UserSignedUp
+        namespace: com.company
+        doc: User sign-up information
+        fields:
+          - name: user-id
+            type: int
+          - name: user-email
+            type: string
+```
+
+In the above snippet: 
+- The `userSignedUp` message is defined with Avro Schema, using the specified `schemaFormat` and the `schema`.
+- The `schema` includes a `record` named `UserSignedUp` within the `com.company` namespace. It also describes two fields, `user-id` and `user-email`, defining their data types as `int` and `string` respectively.
+
+By combining the Avro Schema discussed above into the previous tutorial, you'll have a AsyncAPI document fully equipped with Avro Schema!
+
+```
+asyncapi: 3.0.0
+info:
+  title: User Signup API
+  version: 1.0.0
+  description: The API notifies you whenever a new user signs up in the application.
+servers:
+  kafkaServer:
+    host: test.mykafkacluster.org:8092
+    description: Kafka Server
+    protocol: kafka
+operations:
+  onUserSignedUp:
+    action: receive
+    channel:
+      $ref: '#/channels/userSignedUp'
+channels:
+  userSignedUp:
+    description: This channel contains a message per each user who signs up in our application.
+    address: user_signedup
+    messages:
+      userSignedUp:
+        $ref: '#/components/messages/userSignedUp'
+components:
+  messages:
+    userSignedUp:
+      payload:
+        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+        schema:
+          type: record
+          name: UserSignedUp
+          namespace: com.company
+          doc: User sign-up information
+          fields:
+            - name: user-id
+              type: int
+            - name: user-email
+              type: string
+```
+
+## Defining messages using Avro Schema in JSON
```

Comment:
```suggestion
## Reuse existing Avro Schemas
```

Please modify this section so it sounds like part of story that "when you start using AsyncAPI with existing kafka infra, then you already have existing avro schemas that you just want to reuse from file"